libs/blocks/marquee/marquee.js

```diff
@@ -114,4 +114,13 @@ export default function init(el) {
       media?.lastChild.remove();
     }
   }
+  if (el.classList.contains('mnemonic-list')) {
+    import('./mnemonic-list.js')
+      .then((module) => {
+        module.decorateMnemonicList(foreground);
```

Comment:
I think the `init` method should be `async` and you should `await` the import here.
If someone else adds a more processing after your changes, it can cause unpredictable issues.

Reply:
Thank you, it's addressed.